lib/active_triples/rdf_source.rb

```diff
@@ -90,6 +91,8 @@ def type_registry
     # @see RDF::Graph
     # @todo move this logic out to a Builder?
     def initialize(*args, &block)
+      @observers   = Set.new
```

Comment:
I wonder if this should be a different object.  e.g. `RdfSource#observers.notify`